docs/apache-airflow/administration-and-deployment/logging-monitoring/metrics.rst

```diff
@@ -223,6 +223,7 @@ Name                                                Description
 ``dag_processing.total_parse_time``                 Seconds taken to scan and import ``dag_processing.file_path_queue_size`` DAG files
 ``dag_processing.file_path_queue_size``             Number of DAG files to be considered for the next scan
 ``dag_processing.last_run.seconds_ago.<dag_file>``  Seconds since ``<dag_file>`` was last processed
+``dag_processing.last_num_of_db_queries.<dag_file>``Number of queries to Airflow database during parsing per ``<dag_file>``
```

Comment:
missing whitespace (I guess)
```suggestion
``dag_processing.last_num_of_db_queries.<dag_file>`` Number of queries to Airflow database during parsing per ``<dag_file>``
```

Reply:
I am not sure about additional whitespace, because this text should be part of a table. And I think if we add a whitespace it can broke this table view https://airflow.apache.org/docs/apache-airflow/stable/administration-and-deployment/logging-monitoring/metrics.html#metric-descriptions